app/views/shared/eligible_region_content_components/_professional_recognition.html.erb

```diff
@@ -36,8 +11,15 @@
     <p class="govuk-body">
       You cannot send this document yourself – instead, you must contact <%= region_teaching_authority_name_phrase(region) %> and ask them to send it directly to us at <a class="govuk-link" href="mailto:<%= t("service.email.verification") %>"><%= t("service.email.verification") %></a>.
     </p>
+  <% else %>
+    <p class="govuk-body">
+      You’ll need to provide <%= region_certificate_phrase(region) %>, which you can get by contacting:
+    </p>
+    <% if region.sanction_check_none? && region.status_check_written?%>
+     <%= render "shared/teaching_authority_contact_information", region: %>
+    <% end %>
```

Comment:
```suggestion
    <%= render "shared/teaching_authority_contact_information", region: %>
```

I think we should always render this but change where the "This must be dated within 3 months of you applying for QTS" sentence goes and move it out of this shared partial.